src/app/components/DotMenu.tsx

```diff
@@ -19,13 +19,22 @@ export const DotMenuIcon = styled(EllipsisV)`
 `;
 
 // tslint:disable-next-line:variable-name
-export const DotMenuToggle = styled(React.forwardRef<HTMLDivElement>((props, ref) =>
-  <PlainButton {...props} ref={ref}>
-    <div tabIndex={-1}>
-      <DotMenuIcon />
-    </div>
-  </PlainButton>
-))`
+export const DotMenuToggle = styled(
+  React.forwardRef<HTMLDivElement>((props, ref) => {
+    const {isOpen} = props;
+
+    return (
+      <PlainButton {...{
+        'aria-label': 'Actions',
+        'aria-expanded': isOpen,
```

Comment:
you don't have to embed these in the `props` spread, you can define them as normal react props either before or after the `{...props}` depending on which way you want to override